x/gov/client/cli/prompt.go

```diff
@@ -53,6 +53,8 @@ var suggestedProposalTypes = []proposalType{
 // Prompt prompts the user for all values of the given type.
 // data is the struct to be filled
 // namePrefix is the name to be displayed as "Enter <namePrefix> <field>"
+// TODO: when bringing this in autocli, use proto message instead
```

Comment:
purely cherry-picking the PR, this will get deleted when cherry-picking https://github.com/cosmos/cosmos-sdk/pull/22775.